_config/config.yml

```diff
@@ -4,15 +4,19 @@ Name: foxystripeconfig
 SilverStripe\Security\Member:
   extensions:
     - Dynamic\FoxyStripe\ORM\CustomerExtension
-Page:
+
+PageController:
   extensions:
     - Dynamic\FoxyStripe\ORM\FoxyStripePageExtension
+
 SilverStripe\Forms\DateField:
   default_config:
     showcalendar: true
```

Comment:
Unrelated to the PR, but I think this can go away as the new SS4 field doesn't support turning off the calendar via config.

Reply:
This can be done in a diff PR.

Reply:
i'll grab it while I'm in there